test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -551,5 +551,22 @@ public async Task AddProductDefaultPKAndDifferentColumnOrderTest(SupportedLangua
             await this.SendOutputGetRequest("addproductdefaultpkanddifferentcolumnorder", null, TestUtils.GetPort(lang, true));
             Assert.Equal(1, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithDefaultPK"));
         }
+
+        /// <summary>
+        /// Tests that when using a table with column names having slash (/ or \) in them 
+        /// we can insert data using output binding.
+        /// Excluding C#, JAVA and CSX languages since / or \ are reserved characters and are discouraged to use in variable names
+        /// </summary>
+        [Theory]
+        [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.CSharp, SupportedLanguages.OutOfProc, SupportedLanguages.Java, SupportedLanguages.Csx)]
+        public async Task AddProductWithSlashInColumnName(SupportedLanguages lang)
+        {
+            Assert.Equal(0, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithSlashInColumnNames"));
+            this.StartFunctionHost("AddProductWithSlashInColumnName", lang);
+            await this.SendOutputPostRequest("addproduct-slashcolumns", "");
```

Comment:
Normally we create the object here and pass it in the POST request (which is typically how people will be using output bindings). Is there a reason you chose to do it in the test functions themselves directly instead?

Reply:
I just followed the pattern we had for all the negative scenario tests. All of those have the data hardcoded in the function (not sure why we did that to begin with). But I agree with what you said, will add one more insert and test both.